template/flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  description = "virtual environments";
+
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, flake-utils, devshell, nixpkgs }:
+    flake-utils.lib.eachDefaultSystem (system: {
```

Comment:
I wonder if devshell could provide this boilercode as well?

Reply:
I thought about doing the system logic manually. Just figured making users aware of flake-utils would be a good thing, if they are not already.